src/components/wallet/PageTransactions.vue

```diff
@@ -126,19 +136,18 @@ const txFields = `
 export default {
   name: `page-transactions`,
   components: {
-    TransactionList,
+    TransactionItem,
+    EventList,
     DataEmptyTx,
     TmDataMsg,
     TmPage,
   },
   data: () => ({
-    showing: 15,
     pageNumber: 0,
     validators: [],
```

Comment:
Not sure, but shouldn't this be an object instead of array as we are assigning the validatorMap?